packages/shared/src/components/ShareOptionsMenu.tsx

```diff
@@ -89,7 +89,7 @@ export default function ShareOptionsMenu({
   if (hasSquadAccess && !post?.private) {
     if (!squads?.length) {
       shareOptions.push({
-        icon: <MenuIcon Icon={SquadIcon} />,
+        icon: <MenuIcon Icon={SquadIcon} size="medium" />,
```

Comment:
Don't think this adds anything, the `MenuIcon` always renders a icon in medium size :)